src/utils.js

```diff
@@ -168,12 +169,20 @@ function writeExtraEntry(name, {cjs, esm}, clean = true) {
   )
 }
 
+function hasLocalConfig(moduleName, searchOptions = {}) {
```

Comment:
doesn't prettier have a function for finding config? Maybe we should use that instead?

Reply:
Yes, it does. 

I was thinking that this eventually this could be used for the other commands as well.